x-pack/plugins/apm/public/components/shared/PropertiesTable/__test__/__snapshots__/NestedKeyValueTable.test.tsx.snap

```diff
@@ -133,37 +133,15 @@ exports[`FormattedValue component should render an object 1`] = `
 `;
 
 exports[`FormattedValue component should render null 1`] = `
-.c0 {
-  color: #98A2B3;
-}
-
-<FormattedValue
-  value={null}
->
-  <styled.span>
-    <span
-      className="c0"
-    >
-      N/A
-    </span>
-  </styled.span>
-</FormattedValue>
+<styled.span>
+  N/A
+</styled.span>
```

Comment:
Why did this change?

Reply:
I'm sorry, I forgot to update the snapshot. I've just added a commit with fix.